Mesh_3/doc/Mesh_3/CGAL/Mesh_3/parameters.h

```diff
@@ -451,6 +451,72 @@ unspecified_type odt(const Named_function_parameters& np = parameters::default_v
 template <class NamedParameters = parameters::Default_named_parameters>
 unspecified_type perturb(const Named_function_parameters& np = parameters::default_values());
 
+/*!
+ * \ingroup PkgMesh3Parameters
+ *
+ * The function `parameters::initial_points_generator()` enables the user to
+ * specify a functor following the `InitialPointsGenerator` concept
+ * to the mesh generation function `make_mesh_3()`.
+ * The functor will be called for initialization of the meshing process.
+ * If this parameter is specified without argument, the default behaviour is executed,
+ * i.e. the domain's `construct_initial_points_object()`
+ * is called for the initialization of the meshing process.
+ *
+ * \tparam InitialPointsGenerator a functor following the `InitialPointsGenerator` concept
+ *
+ *  @param generator an instance of the InitialPointsGenerator functor
+ *
+ * \cgalHeading{Example}
+ *
+ * \code{.cpp}
+ * // Mesh generation from labelled image with connexity checks.
+ * C3t3 c3t3 = make_mesh_3<c3t3>(domain,
+ *                               criteria,
+ *                               parameters::initial_points_generator(CGAL::Construct_initial_points_labeled_image(image)));
+ * \endcode
+ *
+ * \sa `CGAL::parameters::initial_points()`
+ * \sa `CGAL::make_mesh_3()`
+ * \sa `MeshDomain_3::Construct_initial_points`
+ *
+ */
+template <typename InitialPointsGenerator>
+unspecified_type initial_points_generator(const InitialPointsGenerator& generator);
+/*!
+ * \ingroup PkgMesh3Parameters
+ *
+ * The function `parameters::initial_points()` enables the user to
+ * specify a `std::vector` of initial points
```

Comment:
It would be possible in two ways, but with drawbacks :
 - By templating `parameters::internal::Initial_points_generator_options` with the user's container type. In this method, `init_c3t3`, `C3t3_initializer` and `make_mesh_3_impl` would need to be templated with the container type as well, like it was in one of the previous commit f2af65444d10eaa3164f5bee189467e9c203445f
 - By copying the user's container into a std::vector. This would force an allocation of the same size as the number of initial points.